heron/tools/tracker/src/python/routers/metrics.py

```diff
@@ -119,6 +119,7 @@ async def get_metrics( # pylint: disable=too-many-arguments
   )
 
 
+@router.get("/metricstimeline", response_model=metricstimeline.MetricsTimeline)
 @router.get("/metrics/timeline", response_model=metricstimeline.MetricsTimeline)
```

Comment:
Sorry, I am not familiar with the `FastAPI` library and I have a general question about it: will this attempt to scrape both endpoints? If so, I think one of them will always return a 404 depending on the case, this will not be an issue? I tried searching and could not find any details.

Reply:
Interesting. I didn't realize this was a feature. According to [this ticket](https://github.com/tiangolo/fastapi/discussions/4299#discussioncomment-1839922), it should work. it also mentions a `deprecated=True` flag that might be a good idea to add.

Reply:
Nice find, very handy.